rfc/github-org-migration.md

```diff
@@ -0,0 +1,773 @@
+    - It SHOULD NOT be migrated.
+- [ ] [broker](https://github.com/web3-storage/broker)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [web3-schema](https://github.com/web3-storage/web3-schema)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [example-forum-dapp](https://github.com/web3-storage/example-forum-dapp)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [fauna-dump](https://github.com/web3-storage/fauna-dump)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [db-migration-pipeline](https://github.com/web3-storage/db-migration-pipeline)
+    - Related to fauna-dump.
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [fauna-dumpify](https://github.com/web3-storage/fauna-dumpify)
+    - Old DB migrations for old products;
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [admin.storage](https://github.com/web3-storage/admin.storage)
+    - Admin web interface for the old api.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [hydra-booster](https://github.com/web3-storage/hydra-booster)
+    - This is a fork.
+    - It SHOULD be deleted.
+- [ ] [ipns-publisher](https://github.com/web3-storage/ipns-publisher)
+    - Create your IPNS records in Javascript and publish them on the IPFS network.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [specs](https://github.com/web3-storage/specs)
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [consul-cluster-go-ipfs](https://github.com/web3-storage/consul-cluster-go-ipfs)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [repin-unpinned-cids](https://github.com/web3-storage/repin-unpinned-cids)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [parse-link-header](https://github.com/web3-storage/parse-link-header)
+    - Old but it should be left be.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [js-replica](https://github.com/web3-storage/js-replica)
+    - Empty repo.
+    - It SHOULD be deleted.
+- [ ] [w3up-archived](https://github.com/web3-storage/w3up-archived)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [upload-protocol](https://github.com/web3-storage/upload-protocol)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [ucanto](https://github.com/web3-storage/ucanto)
+    - UCAN encoding implication for creating UCANs and sending them to a service and verifying delegations.
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [w3up](https://github.com/web3-storage/w3up)
+    - Implementation of the protocols and specs repos. The current web3.storage system.
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [ucanto-name-system](https://github.com/web3-storage/ucanto-name-system)
+    - [ ] check with @gobengo what should be done
```

Comment:
I don't know of anything that relies on or depends on this. It would be nice if that URL stays resolving. I think that means a archive or migration to other org is safe.

Reply:
Archiving is probably the safest, so I'll suggest that